rfqmsg/messages.go

```diff
@@ -90,6 +90,63 @@ func (id *ID) Record() tlv.Record {
 	return tlv.MakeStaticRecord(0, id, recordSize, IdEncoder, IdDecoder)
 }
 
+// TransferType defines the type of transaction which will be performed if the
+// quote request leads to an accepted agreement.
+type TransferType uint8
+
+const (
+	// UnspecifiedTransferType represents an undefined or transfer type.
+	UnspecifiedTransferType TransferType = iota
```

Comment:
Constants that are stored in the DB or are transferred over the network should never use `iota`, otherwise it's too easy to accidentally add a value somewhere in the middle and changing the meaning of all following constants.